packages/core/src/spec/types/DeviceTransport.ts

```diff
@@ -2,6 +2,9 @@ import { z } from "zod";
 
 const schemaDeviceTransportParams = z.object({
   appName: z.string().optional(),
+  appVersion: z.string().optional(),
```

Comment:
Improvement, we could use a regex for the semver string
cf. the one recommended on [semver.org](https://semver.org) -> https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

Reply:
fixed